ChangeLog

```diff
@@ -53,6 +53,11 @@ Release date: TBA
 
   Closes #4698
 
+* Fix false-positive of ``unused-private-member`` when using ``__new__`` in a class
+
+  Closes #4668
+
+
```

Comment:
```suggestion

```
Just one empty line :)
So that there are two empty ones between the last entry and the `What's New`.